drivers/janitza_umg96rm.json

```diff
@@ -1,38 +1,39 @@
-        "freq": {"datatype": "float", "description": "Frequency", "multiplier": 1, "register": 19050, "unit": "Hz"},
-        "I_L1": {"datatype": "float", "description": "Phase 1 current", "multiplier": 1, "register": 19012, "unit": "A"},
-        "I_L2": {"datatype": "float", "description": "Phase 2 current", "multiplier": 1, "register": 19014, "unit": "A"},
-        "I_L3": {"datatype": "float", "description": "Phase 3 current", "multiplier": 1, "register": 19016, "unit": "A"},
-        "P_L1": {"datatype": "float", "description": "Phase 1 active power", "multiplier": 1, "register": 19020, "unit": "W"},
-        "P_L2": {"datatype": "float", "description": "Phase 2 active power", "multiplier": 1, "register": 19022, "unit": "W"},
-        "P_L3": {"datatype": "float", "description": "Phase 3 active power", "multiplier": 1, "register": 19024, "unit": "W"},
-        "P_total": {"datatype": "float", "description": "Total active power", "multiplier": 1, "register": 19026, "unit": "W"},
-        "pf_L1": {"datatype": "float", "description": "Phase 1 power factor", "multiplier": 1, "register": 828},
-        "pf_L2": {"datatype": "float", "description": "Phase 2 power factor", "multiplier": 1, "register": 830},
-        "pf_L3": {"datatype": "float", "description": "Phase 3 power factor", "multiplier": 1, "register": 832},
-        "Q_L1": {"datatype": "float", "description": "Phase 1 reactive power", "multiplier": 1, "register": 19036, "unit": "VAr"},
-        "Q_L2": {"datatype": "float", "description": "Phase 2 reactive power", "multiplier": 1, "register": 19038, "unit": "VAr"},
-        "Q_L3": {"datatype": "float", "description": "Phase 3 reactive power", "multiplier": 1, "register": 19042, "unit": "VAr"},
-        "Q_total": {"datatype": "float", "description": "Total reactive power", "multiplier": 1, "register": 19044, "unit": "VAr"},
-        "S_L1": {"datatype": "float", "description": "Phase 1 apparent power", "multiplier": 1, "register": 19028, "unit": "VA"},
-        "S_L2": {"datatype": "float", "description": "Phase 2 apparent power", "multiplier": 1, "register": 19030, "unit": "VA"},
-        "S_L3": {"datatype": "float", "description": "Phase 3 apparent power", "multiplier": 1, "register": 19032, "unit": "VA"},
-        "S_total": {"datatype": "float", "description": "Total apparent power", "multiplier": 1, "register": 19034, "unit": "VA"},
-        "V_L1": {"datatype": "float", "description": "Phase 1 line to neutral volts", "multiplier": 1, "register": 19000, "unit": "V"},
-        "V_L2": {"datatype": "float", "description": "Phase 2 line to neutral volts", "multiplier": 1, "register": 19002, "unit": "V"},
-        "V_L3": {"datatype": "flaot", "description": "Phase 3 line to neutral volts", "multiplier": 1, "register": 19004, "unit": "V"}
+        "E_exported_Total": { "description": "total export energy", "register": 19076, "unit": "Wh"},
```

Comment:
I meant small letter in the variable name, not in the description :). (FYI the description is for documentation purposes only and not used elsewhere)

Reply:
Hahaha I think I'm still a little bit sleepy :)

Reply:
It's ok let's check with Olu first before making any other changes.

Reply:
So we had a talk with Olu and we come to the decision that since we are keeping all the energy readings the name of the imported and exported field can be chosen either with or without the "total" but I think that, since we are still testing this field, it can be clearer to add "total" to make sure we are not using it for now.

Reply:
Ok that makes sense. It gets controlled ultimately in the datamap anyway.

Let's go ahead and test this on the datalogger (by putting it in the config).